HabitTracker/Report.cs

```diff
@@ -0,0 +1,149 @@
+﻿namespace HabitTracker
+{
+    public class Report
+    {
+        enum Frequency
+        {
+            Daily,
+            Weekly,
+            Monthly
+        }
+        static public void GetReport()
+        {
+            var connection = Data.DbConnection.GetConnection();
```

Comment:
🟠 **Database Connections**

💡 I would re-think your db connection strategy/pattern. Connections should be initialised with a using statement to dispose and manage safely. You don't seem to ever close this connection either (which you wouldn't have to worry about with a using statement).

🔧 Use using statements with connections. Reduce the amount of time a connection is open.